python/tvm/relay/frontend/tflite.py

```diff
@@ -3210,7 +3210,7 @@ def get_pad_value(data, kernel, stride):
     """
 
     out = int(math.ceil(float(data) / float(stride)))
-    pad = max(0, (out - 1) * stride + kernel - data)
+    pad = max(0, (out - 1) * int(stride) + int(kernel) - int(data))
```

Comment:
Do we want to somehow test the compilation for skylake/cascadelake target?

Reply:
Also, do we know why this fails with `int64`? AFAIU from your RFC: https://discuss.tvm.apache.org/t/int64-vs-int32-dtype-error/8555 this used to work before.